app/components/public/social-links.js

```diff
@@ -9,5 +10,13 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+   calender:computed('eventtime','eventtimeends','eventname','eventlocation',function(){
+   let startparams=this.get('eventtime');
+   let endparams=this.get('eventtimeends');
+   const starttime=moment(startparams).tz("Asia/Kolkata").utc().format('YYYYMMDD[T]HHmmSS[Z]');
+   const endtime=moment(endparams).tz("Asia/Kolkata").utc().format('YYYYMMDD[T]HHmmSS[Z]');
+   return `https://calendar.google.com/calendar/u/0/r/eventedit?dates=${starttime}/${endtime}&text=${this.get('eventname')}&location=${this.get('eventlocation')}&sf=true`;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)